app/services/notifications/setup/submit-ad-hoc-licence.service.js

```diff
@@ -45,6 +46,34 @@ async function go(sessionId, payload) {
   return {}
 }
 
+/**
+ * Ad-hoc licence notifications has highlighted an issue with using the returns period dates.
+ *
+ * This code is intended as a temporary 'fix' to allow ad-hoc returns to use the returns invitations Notify template
+ * (which requires the 'determinedReturnsPeriod' dates)
+ *
+ * This function alters the 'dueDate' by adding 28 days to the current date (this may not be the final state and should be regarded as a
+ * placeholder).
+ *
+ * This also, manually, sets gets the 'startDate' and 'endDate' for 'allYear' which represents the current financial
+ * kyear.
+ *
+ * @private
+ */
+function _determinedReturnsPeriod() {
+  const { returnsPeriod, summer } = DetermineReturnsPeriodService.go('allYear')
+
+  const dueDate = new Date()
+  const daysToAdd = 28
+  dueDate.setDate(dueDate.getDate() + daysToAdd)
+
+  return {
+    ...returnsPeriod,
+    summer,
+    dueDate
+  }
+}
```

Comment:
Perhaps...?

```suggestion
  const dueDate = new Date()
  
  dueDate.setDate(dueDate.getDate() + 28)
  
  // TODO: Remove this use of DetermineReturnsPeriodService(). Ad-hoc will use the same Notify template as invitations,
  // but currently it expects period start and end date values to be provided. We don't have these on the ad-hoc journey
  // so for now are using whatever the current return period is. Once we have confirmation the template has been updated
  // we can drop this call. 
  const returnsPeriod = DetermineReturnsPeriodService.go('allYear')

  return {
    dueDate,
    ...returnsPeriod
  }
}
```

Reply:
This will still need summer for the return as it will be set to false to not affect the legacy code queries.